src/test/java/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinitionTest.java

```diff
@@ -1,47 +0,0 @@
-package org.jenkinsci.plugins.workflow.cps;
-
-import org.junit.Test;
-
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.assertEquals;
-
-/**
- * @author Kohsuke Kawaguchi
- */
-public class CpsFlowDefinitionTest extends AbstractCpsFlowTest {
```

Comment:
Yes these tests date to the first days of Workflow and are not very useful now.

Worth checking whether `FlowExecution.getCauseOfFailure` is covered anywhere else, though I am not even sure if anyone calls it anyway.

Reply:
I went ahead and added coverage for `FlowExecution.getCauseOfFailure` to a generic sandbox test in ea3cd8a since it didn't look like it was used anywhere else.